src/node_contextify.cc

```diff
@@ -199,7 +199,9 @@ MaybeLocal<Context> ContextifyContext::CreateV8Context(
       object_template,
       {},       // global object
       {},       // deserialization callback
-      microtask_queue() ? microtask_queue().get() : nullptr);
+      microtask_queue() ?
+          microtask_queue().get() :
+          env->isolate()->GetCurrentContext()->GetMicrotaskQueue());
```

Comment:
env->context()?

Reply:
That's equivalent right now, yes, but I'd like to write this in a more future-proof way if you don't mind :)

Reply:
There will be situations where the active environment doesn't match the active context?

Reply:
Well, we could extend Node.js to supporting multiple different Contexts per Environment, which would be nice, and we’re not *that* far from supporting it